python/docs/source/reference/pyspark.ml.rst

```diff
@@ -351,6 +351,7 @@ Distributor
     :toctree: api/
 
     TorchDistributor
+    DeepspeedTorchDistributor
```

Comment:
Seems like it fails:

```
2023-07-12T23:31:59.4320534Z Warning, treated as error:
2023-07-12T23:31:59.4322346Z [autosummary] failed to import 'pyspark.ml.torch.distributor.DeepspeedTorchDistributor': no module named pyspark.ml.torch.distributor.DeepspeedTorchDistributor
```

Reply:
oh it's torch.deepspeed.DeepspeedTorchDistributor